benches/diesel_1_1_1/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "diesel_1_1_1"
 version = "0.1.0"
-authors = ["Adam Perry <adam.n.perry@gmail.com>"]
+license = "MIT OR Apache-2.0"
```

Comment:
In other `Cargo.toml`s you're doing this with `/` instead of `OR`

Reply:
Aha! I just copied from the cargo manifest for each crate.